src/main/java/com/rybka/model/BankData.java

```diff
@@ -0,0 +1,16 @@
+package com.rybka.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+@AllArgsConstructor
+public class BankData {
+    private String name;
+    private String link;
+    private double buy;
+    private double sell;
+    private Date lastUpdated;
```

Comment:
Use LocalDateTime it's far more convenient to work with it then Date object.

Reply:
fixed